app2/src/main/java/com/example/recipebox/MainActivity.kt

```diff
@@ -98,28 +99,43 @@ fun Scaffold() {
 				}
 			)}
 
-	) { padding ->
-			Column(modifier = Modifier.padding(10.dp)) {
+	) {
+		Column(modifier = Modifier.padding(10.dp)) {
 				OutlinedTextField(
 					value = recipeName,
 					onValueChange = { recipeName = it },
 					label = { Text(stringResource(R.string.recipeName)) },
-					modifier = Modifier.fillMaxWidth()
+					modifier = Modifier.fillMaxWidth(),
 				)
 				Button(
 					onClick = {
-						Toast.makeText(context, "$recipeName", Toast.LENGTH_LONG).show()
+						Toast.makeText(context, recipeName, Toast.LENGTH_LONG).show()
 					}
 				) {
 					Text(text = "Search", modifier = Modifier.fillMaxWidth())
 				}
 
 				//Temporary
-				Text(text="Recipe 1", modifier = Modifier.background(MaterialTheme.colors.secondary).fillMaxWidth().padding(vertical = 50.dp))
-				Text(text="Recipe 2", modifier = Modifier.background(MaterialTheme.colors.secondary).fillMaxWidth().padding(vertical = 50.dp))
-				Text(text="Recipe 3", modifier = Modifier.background(MaterialTheme.colors.secondary).fillMaxWidth().padding(vertical = 50.dp))
-				Text(text="Recipe 4", modifier = Modifier.background(MaterialTheme.colors.secondary).fillMaxWidth().padding(vertical = 50.dp))
-				Text(text="Recipe 5", modifier = Modifier.background(MaterialTheme.colors.secondary).fillMaxWidth().padding(vertical = 50.dp))
+				Text(text="Recipe 1", modifier = Modifier
```

Comment:
I would not commit formatting changes since your IDE can do it and it does not reduce technical dept. However, I committed it so it would be more readable for me while I review.

Reply:
Yes, good point.